rust/operator-binary/src/config.rs

```diff
@@ -183,7 +194,7 @@ impl HdfsSiteConfigBuilder {
                 format!("{}.{}.{}", address, self.logical_name, nn.pod_name),
                 format!(
                     "{}:{}",
-                    nn.fqdn(),
+                    nn.fqdn(cluster_info),
```

Comment:
Just a nit: lets do it consistent and use the identifiers instead of relying on order?